binderhub/launcher.py

```diff
@@ -35,6 +35,12 @@ class Launcher(LoggingConfigurable):
     hub_api_token = Unicode(help="The API token for the Hub")
     hub_url = Unicode(help="The URL of the Hub")
     create_user = Bool(True, help="Create a new Hub user")
+    allow_named_servers = Bool(
+        os.getenv('JUPYTERHUB_ALLOW_NAMED_SERVERS', "false") == "true",
```

Comment:
It feels like there should be a standard way of converting an env-var to a bool, for instance to handle `False|True|0|1` etc which might be an issue if binderhub is run independently of the helm chart. 

I'm not aware of anything though, and since the helm chart should always pass `true|false` this is fine for now unless anyone else has an quick fix.

Reply:
I wish for a solution to this as well (in particular in other projects at work) but I've never been able to find anything ;(